README.md

```diff
@@ -38,6 +38,9 @@ matrix below:
 
 | Triton CLI Version | TRT-LLM Version | Triton Container Tag |
 |:------------------:|:---------------:|:--------------------:|
+| 0.0.11 | v0.15.0 | 24.11 |
```

Comment:
Do we want to copy over the `convert_checkpoints.py` scripts for trtllm version 0.13.0 and 0.14.0 and release a new version of the CLI with the updated trtllm templates scripts being the main difference?